root/edit/details/AddReleaseGroup.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+
+type AddReleaseGroupProp = {
+  ...EditT,
+  +display_data: {
+      +artist_credit: ArtistCreditT,
```

Comment:
This block is indented a couple spaces too many
```suggestion
    +artist_credit: ArtistCreditT,
```